livechat/utils/structures.py

```diff
@@ -30,3 +33,19 @@ def success(self) -> bool:
     def payload(self) -> dict:
         ''' `payload` from the RTM response. '''
         return self.rtm_response.get('payload')
+
+
+class Scheme(Enum):
```

Comment:
also, it's a top-level definition without a direct link to the token, so i would consider `TokenType` or sth. importing `from livechat.utils.structures import Scheme` does not clearly indicate what scheme we are actually dealing with